src/main/kotlin/command/Remind.kt

```
@@ -56,22 +61,30 @@ class Remind(private val jda: JDA) : Command() {
         }
         reminders?.forEach {
             val user = it.user
-            val channel = jda.retrieveUserById(user.id).complete()?.openPrivateChannel()?.complete()
-            if (it.reminder.length > 2000) {
+            val channel = try { jda.retrieveUserById(user.id).await()?.openPrivateChannel()?.await() } catch (e: Exception) { null } // Can't throw exception here as this isn't a command handler
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)